### DISCUSSION TITLE
Manual Poster Cropping From Generated Artwork

### DISCUSSION THREAD

**Discussion by IcySon55** [gotson/komga#1528] [Ideas]
### Describe your suggested feature

It would be very useful to be able to crop the existing poster image that was generated from the CBR, CBZ, etc...

Note: This is not a request for automatic cropping, but rather for a UI tool to manually crop the generated poster image.

Often some books have the cover as 2 pages wide and cropping to only the cover would be nice.

Example:
![4-14-2023 3-09-16 AM](https://user-images.githubusercontent.com/4888934/231970034-ea184099-cb40-4b68-aed7-20f48849f7f9.jpg)

An example tool that could provide the functionality: https://fengyuanchen.github.io/cropperjs/

Thanks!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
You can use custom cover for those corner cases.

**Comment by IcySon55** [comment]
Sure, but then I have to manually create the covers, or even add an extra page of just the cover already edited to the CBZ. Much less convenient file management-wise than being able to crop within Komga...

**Reply by WiiGe** [reply]
This function is useful for relative large library, let's say 10000+ series manga with upload size limit(for faster web loading):

1. you have to download cover 
2. open image processing app and corp it and 
3. resize to size limit(maybe 1MB or 5MB) 
4. then upload it.

not very convenient for library management so far.  One button corp & save refine this process a lot.
